docs/src/app_dev/app_val_tokenomics/cc_xfer_splice_wallet_nomics.rst

```diff
@@ -0,0 +1,59 @@
+..
+   Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+..
+   SPDX-License-Identifier: Apache-2.0
+
+.. _cc_xfer_splice_wallet_nomics:
+
+A CC Transfer using the Splice Wallet UI
+========================================
+
+A manual CC transfer using the UI of the Splice wallet (the wallet UI
+built into every validator) is an example of a :ref:`cc_xfer_nomics`.
+There are three types of manual transfers possible with the Splice
+wallet UI:
+
+      -  *Amulet legacy transfer* offer where the receiver has to accept the transfer offer and then automation on the sender’s side completes the transfer. The ``provider`` party on the
+         `AppRewardCoupon <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet.html#type-splice-amulet-apprewardcoupon-57229>`__
+         is the ``sender``. The ``sender`` is assigned to the `ValidatorRewardCoupon <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet.html#type-splice-amulet-validatorrewardcoupon-76808>`__
+         user field. These transfers are never featured.
+
+      -  A two-step, *CN token standard based*
+         `Transfer <https://docs.dev.sync.global/app_dev/api/splice-api-token-transfer-instruction-v1/Splice-Api-Token-TransferInstructionV1.html#type-splice-api-token-transferinstructionv1-transfer-51973>`__
+         offer first locks the desired CC amount; on acceptance, it is
+         unlocked and the transfer is completed. This does not rely on
+         automation and works well for external parties. Note that there
+         the two steps show up in the wallet, with each step having fees
+         that are rewarded:
```

Comment:
We are planning to propose a CIP to remove all usage fees, which would mean that no activity is recorded for these kinds of transfers as they are not featured.

It would be good to add a disclaimer to the documentation somewhere calling out the the usage fees are small and might be removed, compared to the gains from being a featured app.